packages/react-shopify-app-bridge/src/Provider.tsx

```diff
@@ -219,51 +186,39 @@ export const Provider = ({
     });
   }, []);
 
-  let app = (
-    <GadgetUrqlProvider api={api}>
-      <InnerGadgetProvider
-        forceRedirect={forceRedirect}
-        isEmbedded={isEmbedded}
-        gadgetAppUrl={gadgetAppUrl}
-        api={api}
-        originalQueryParams={originalQueryParams}
-        isRootFrameRequest={isRootFrameRequest}
-        type={coalescedType}
-        isInstallRequest={!!isInstallRequest}
-      >
-        {children}
-      </InnerGadgetProvider>
-    </GadgetUrqlProvider>
-  );
-
-  const shouldMountAppBridge = host && coalescedType != AppType.Standalone && (!isInstallRequest || inDestinationContext);
-
   console.debug("[gadget-rsab] provider rendering", {
     host,
     coalescedType,
-    isInstallRequest,
     isReady,
-    isEmbedded,
-    isRootFrameRequest,
-    inDestinationContext,
-    shouldMountAppBridge,
   });
 
-  // app bridge provider seems to prevent urql from sending graphql requests when it cannot communicate using postMessage when not embedded so we must skip using the app bridge provider on the very first redirect from shopify
-  if (shouldMountAppBridge) {
-    app = (
-      <AppBridgeReact.Provider
-        config={{
-          apiKey: shopifyApiKey,
-          host,
-          forceRedirect,
-        }}
-        router={router}
-      >
-        {app}
-      </AppBridgeReact.Provider>
-    );
-  }
-
-  return app;
+  return (
+    <GadgetUrqlProvider api={api}>
+      {shopifyGlobalDefined ? (
```

Comment:
will this handle the case where the app type is standalone? I think we need to check that here

Reply:
ah yea, standalone app check should be added here too

Reply:
i updated this to render the plain children if you have a standalone app. it seems like the old way was to try and redirect you to gadget which would then send you to shopify and then you'd end up in a redirect loop which seemed wrong